docs/source/users.md

```diff
@@ -70,6 +70,17 @@ These exception objects are not raised so that the generator may keep parsing. I
 are yielded from the generator with a `partial_data` attribute for user examination. This partial data allows you to 
 see how far it got through a container inheritance structure before failing to determine the rest of the structure. 
 
+## Common Issues and Solutions
+
+### Parser Generator Completes without Yielding a Packet
+This can occur if your data file contains only packets that do not match any packet definitions in your XTCE document 
+and `yield_unrecognized_packet_errors=False` (the default). This could mean that your data file actually contains only 
+APIDs that are not covered in your packet definition but usually it means you have bad restriction criteria.
```

Comment:
Is there a concrete example for what "bad restriction criteria" means as well? I think spelling these things out explicitly with one example will allow people to generalize easier.